src/Host.cpp

```diff
@@ -1870,3 +1879,13 @@ void Host::loadSecuredPassword()
 
     job->start();
 }
+
+// Only needed for places outside of this class (currently believed to be):
+// TBuffer::decodeOSC(const QString&)
+// TBuffer::resetColors()
+// dlgProfilePreferences::resetColors()
+// dlgProfilePreferences::setColor(QPushButton*, QColor&)
```

Comment:
This will get out of date and be misleading since we won't be updating it... I recommend not listing these things explicitly. Any modern IDE has an easy way to find where is this function used!

Reply:
Okay - I'll delete it; I think I was just trying to keep track myself and to help me try to work out where I hadn't used it but needed to...

Reply:
Done in next commit.